ui/src/Panels/Consumer/Consumer.view.js

```diff
@@ -66,19 +67,29 @@ const Consumer = (props) => {
       </div>
       <ConsumerMessages>
         {messages.map((msg, index) => (
-          <ConsumerMessage
-            {...idAttributeGenerator('consumer_consumed_message')}
-            key={`consumed-message-${index}:${msg.index}`}
-            isFirst={index === 0}
-            error={
-              msg.status === CONSTANTS.VERTX_ERROR_STATUS
-                ? { message: translate('ERROR_CONSUMING', {}, true) }
-                : undefined
-            }
-            message={
-              msg.status !== CONSTANTS.VERTX_ERROR_STATUS ? msg : undefined
-            }
-          />
+          <SelectedMessageConsumer key={`smc-${index}:${msg.index}`}>
+            {({ updateSelectedMessage, isSameAsSelected }) => {
+              return (
+                <ConsumerMessage
+                  {...idAttributeGenerator('consumer_consumed_message')}
+                  key={`consumed-message-${index}:${msg.index}`}
```

Comment:
Does the `ConsumerMessage` still need a key prop if it's no longer a list item?

Reply:
You are right, it probably doesn't at this point - will look into it!